blocks/carousel/carousel.css

```diff
@@ -5,94 +5,99 @@ main .carousel-wrapper {
-        background: #091d2f;
-        color: white;
-        flex-direction: column;
-        align-items: flex-start;
-        border-radius: .333rem;
-        height: 36.667rem;
-        width: 35.111rem;
+        .content-card {
+            display: flex;
+            padding: 5.556rem 7.5rem 0 3.333rem;
+            background: #091d2f;
+            color: white;
+            flex-direction: column;
+            align-items: flex-start;
+            border-radius: .333rem;
+            height: 36.667rem;
+            width: 35.111rem;
 
-        > p {
-          font-size: 2.111rem;
-          line-height: 2.111rem;
-          margin-bottom: 1.667rem;
-          color: #bea265;
-        }
+            .content-title {
+                font-size: 2.111rem;
+                line-height: 2.111rem;
+                margin-bottom: 1.667rem;
+                color: #bea265;
+                font-family: var(--heading-font-family);
+            }
 
-        p:nth-child(2),
-        div[data-aue-label='Text'] p {
-          font-size: 1.111rem;
-          line-height: 1.778rem;
-          color: white;
+            .content-text {
+                font-size: 1.111rem;
+                line-height: 1.778rem;
+                color: white;
+                font-family: var(--body-font-family);
+            }
         }
       }
-    }
+  }
 
-  .carousel-actions {
-    display: inline-flex;
-    gap: .444rem;
-    position: absolute;
-    right: 1.667rem;
-    top: 32rem;
+    .carousel-actions {
+        display: inline-flex;
+        gap: .444rem;
+        position: absolute;
+        right: 1.667rem;
+        top: 32rem;
 
-      .action-prev,
-      .action-next {
-        height: 3rem;
-        padding: .944rem;
-        width: 3rem;
-        backdrop-filter: blur(30px);
-        border: none;
-        border-radius: 50%;
-        display: block;
-        background-color: hsla(41deg 41% 57%/ 85%);
-        position: relative;
-      }
+        .action-prev,
+        .action-next {
+            height: 3rem;
+            padding: .944rem;
+            width: 3rem;
+            backdrop-filter: blur(30px);
+            border: none;
+            border-radius: 50%;
+            display: block;
+            background-color: hsla(41deg 41% 57%/ 85%);
+            position: relative;
+        }
 
-      .action-prev::before,
-      .action-next::before {
-         background-image: url('https://www.oceanfinancial.com/etc.clientlibs/oceanfinancial/clientlibs/clientlib-site/resources/images/ofs-icon-chevron-right-white.svg');
-         background-origin: padding-box;
-         content: '';
-         height: 20px;
-         width: 20px;
-         display: block;
-         background-size: contain;
-     }
+        .action-prev::before,
+        .action-next::before {
+            background-image: url('https://www.oceanfinancial.com/etc.clientlibs/oceanfinancial/clientlibs/clientlib-site/resources/images/ofs-icon-chevron-right-white.svg');
```

Comment:
I think we should try to add these icons as part of the repo itself and get it that way instead of relying on an image in an AEM client library.  Look and see how I did it in the flip card.  You might have to tweak how it works (not a background-image) but this project shouldn't rely on a library not contained within itself.